src/Filesystem/Local.php

```diff
@@ -0,0 +1,102 @@
+<?php
+namespace Bolt\Filesystem;
+
+use League\Flysystem\Adapter\Local as LocalBase;
+use League\Flysystem\Config;
+use League\Flysystem\Util;
+
+class Local extends LocalBase
+{
+    public function __construct($root)
+    {
+        $realRoot = $this->ensureDirectory($root);
+        $this->setPathPrefix($realRoot);
+    }
+
+    protected function ensureDirectory($root)
+    {
+        if (is_dir($root) === false || !is_writable($root)) {
+            return false;
+        }
+
+        return realpath($root);
+    }
```

Comment:
Commenting before coffee doesn't ether…  

What I meant was that if the parent directory **is** writeable, and the directory being *ensured* doesn't exist, we should create it.